test/functional/specs/Config Overrides/C7437530.js

```diff
@@ -0,0 +1,92 @@
+import { t } from "testcafe";
```

Comment:
Can you add a test where the configOverrides are provided on the configure command and the sendEvent command? Make sure they are overridden appropriately.